src/components/books/Page.js

```diff
@@ -1,41 +1,16 @@
+import { useSelector } from 'react-redux';
 import Books from './Books';
 import AddBook from './AddBook';
 
-const books = [
-  {
-    category: 'Action',
-    title: 'The Hunger Games',
-    author: 'Suzanne Collins',
-    progress: {
-      currentChapter: 'Chapter 17',
-      completed: '64',
-    },
-  },
-  {
-    category: 'Science Fiction',
-    title: 'Dune',
-    author: 'Frank Herbert',
-    progress: {
-      currentChapter: 'Chapter 3: "A Lesson Learned"',
-      completed: '8',
-    },
-  },
-  {
-    category: 'Economy',
-    title: 'Capital in the Twenty-First Century',
-    author: 'Suzanne Collins',
-    progress: {
-      currentChapter: 'Introduction',
-      completed: '0',
-    },
-  },
-];
```

Comment:
**Optional Changes**
The former Code Reviewer might have missed this.
- [ ] The initial State should be an empty `array`. By introducing objects in the array has added a layer of complexity to your code at the moment. I guess it is the reason you're finding it a little tedious to implement completed progress in the `BookProgress.js`